.github/workflows/ci.yml

```diff
@@ -0,0 +1,96 @@
+name: CI
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+    workflow_dispatch:
+
+env:
+    APPLICATION_ENV: 'development'
+    APPLICATION_STORE: 'DE'
+
+jobs:
+    validation:
+        runs-on: ubuntu-18.04
+        strategy:
+            fail-fast: false
+            matrix:
+                php-version: [
+                        '7.3',
+                        '7.4',
```

Comment:
where did you get this file? it should be 8.0 here

Reply:
I assume this is an old file from original integrator code

Reply:
Do we actually need this this file? Because even Ubuntu 18.04 and PHP 8.0 will be tricky to use. Or do we need to update this to latest versions?